doc/connectivity/bluetooth/bluetooth-shell.rst

```diff
@@ -0,0 +1,357 @@
+.. _bluetooth_shell:
+
+Bluetooth Shell
+###############
+
+Bluetooth Shell Setup and Usage
```

Comment:
A small sentence explaining what "Bluetooth Shell" is and what it can be used for.